src/test/kotlin/no/nav/tilleggsstonader/sak/vedtak/barnetilsyn/StepDefinitions.kt

```diff
@@ -62,34 +46,20 @@ class StepDefinitions {
 
     @Gitt("følgende støndsperioder")
     fun `følgende støndsperioder`(dataTable: DataTable) {
-        every { stønadsperiodeRepository.findAllByBehandlingId(behandlingId) } returns mapStønadsperioder(dataTable)
+        every { stønadsperiodeRepository.findAllByBehandlingId(behandlingId) } returns mapStønadsperioder(
+            behandlingId,
+            dataTable,
+        )
+    }
```

Comment:
Småpirk men synes dette blir vanskelig å lese. For meg blir ny rad for mapStønadsperioder enklere for meg
```kotlin
every { stønadsperiodeRepository.findAllByBehandlingId(behandlingId) } returns 
                mapStønadsperioder(behandlingId, dataTable)
```